File: app/src/main/java/com/levibostian/androidblanky/activity/MainActivity.java
package com.levibostian.androidblanky.activity;

import android.os.Bundle;
import com.levibostian.androidblanky.R;
import com.levibostian.androidblanky.fragment.MainFragment;

public class MainActivity extends BaseActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setContentView(R.layout.activity_main);

        startupFragment();
    }

    private void startupFragment() {
        getSupportFragmentManager().beginTransaction().add(R.id.fragment_container, MainFragment.newInstance()).commit();
    }

}

File: app/src/main/java/com/levibostian/androidblanky/activity/MainActivity.kt
package com.levibostian.androidblanky.activity

import android.support.v4.app.Fragment
import android.support.v7.app.AppCompatActivity
import com.curiosityio.andoidviews.activity.BaseFragmentActivity
import com.levibostian.androidblanky.fragment.MainFragment

class MainActivity : BaseFragmentActivity() {

    override fun getInitialFragment(): Fragment? {
        return MainFragment.newInstance()
    }

}